app/views/keywords/index.html.slim

```diff
@@ -4,14 +4,17 @@
 
   = render(partial: 'search')
 
-  - if keywords.present?
-    table class="table table-hover my-5 w-50"
+  - if keywords.present?    
```

Comment:
please remove empty spacea after `present?`

Reply:
Fixed in d5684d16e6f21c255cc669a83245943ec64767cb